<?xml version="1.0"?>
<launch>

<arg name="vehicleID" default="tum-q7"/>

<group ns="/Operator/Visual">
    <node pkg="tod_visual" name="tod_visual" type="Interface" output="screen">
        <rosparam command="load" file="$(find tod_vehicle_config)/vehicle_config/$(arg vehicleID)/sensors-camera.yaml" />
        <param name="vehicleID" value="$(arg vehicleID)"/>
        <param name="debug" value="false"/>
        <remap from="primary_control_cmd" to="/Operator/CommandCreation/primary_control_cmd"/>
        <remap from="secondary_control_cmd" to="/Operator/CommandCreation/secondary_control_cmd"/>
    </node>
</group>

</launch>
